wordpress-mobile/WordPress-Android
# Disable my site improvements for everyone

This PR reverts the changes that randomized user distribution of my site improvements experiment and disables it for everyone based on BuildConfig value.

To test:
- Start the app
- Notice the old my site fragment is visible
- Check that you can enable my site improvements in the test feature configuration settings

## Regression Notes
1. Potential unintended areas of impact
- nothing

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- n/a

3. What automated tests I added (or what prevented me from doing so)
- n/a


PR submission checklist:
